Automattic/wp-calypso
# Signup reskin: Remove example suggestions and dropdown filters

#### Changes proposed in this Pull Request

This PR is towards implementing the reskinned signup experiment. For more context on the experiment, read pbxNRc-IO-p2.

NOTE:

This PR implements only the UI update for the example suggestions section. The other changes will be implemented in subsequent PRs.

Here's the [Figma file](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) for reference.

For the record:
Parent PR: https://github.com/Automattic/wp-calypso/pull/51499 and root PR: https://github.com/Automattic/wp-calypso/pull/50512

<img width="1138" alt="Screenshot 2021-03-31 at 5 04 16 PM" src="https://user-images.githubusercontent.com/1269602/113148701-c9c2f180-924f-11eb-8387-8ae79da1a8e5.png">


#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start.
* Proceed to the domains step and verify that the UI before typing anything in the search input box matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536). The things to look out for specifically:
    - The gear icon filter is removed from the search input box
    - The search icon in the input box has an updated fill colour
    - The section below the search input box shows a tip for selecting domain names
* Start over and verify that control group experience is unaffected
